app/Services/UpdatePasswordService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Services;
```

Comment:
Move these services to some new namespace like `Brewmap/Services/User` or something like that.